app/lib/actions/katello/product/content_create.rb

```diff
@@ -8,12 +8,13 @@ def plan(root)
           sequence do
             if root.content_id.nil?
               content_create = plan_action(Candlepin::Product::ContentCreate,
-                                           owner:       root.product.organization.label,
-                                           name:        root.name,
-                                           type:        root.content_type,
-                                           arches:      root.format_arches,
-                                           label:       root.custom_content_label,
-                                           content_url: root.custom_content_path)
+                                           owner:         root.product.organization.label,
+                                           name:          root.name,
+                                           type:          root.content_type,
+                                           arches:        root.format_arches,
+                                           label:         root.custom_content_label,
+                                           os_versions:   root.os_versions&.join(','),
```

Comment:
maybe another find/replace fail?

Reply:
It stays `os_versions` in the Katello action.  It only gets transformed to `requiredTags` in the Candlepin action.  I was guessing that's how you'd want it...

Reply:
Oops, ignore me. I thought that was going into candlepin.